styleguide/STYLE.md

```diff
@@ -0,0 +1,174 @@
+Linter cannot do all of the work for you, so please pay attention to the following cases:
+
+* **Their/they’re/there, your/you’re** - [check](https://www.dictionary.com/e/their-there-theyre/ ) if you’ve got the right one for your situation!  
+* **Affect/effect** - affect is a verb, effect is a noun. When you affect something, you’re impacting or changing the thing. When you have an effect, it’s the outcome or result of a change. 
+* Be mindful of **hyphens**! Hyphens for compound adjectives, no hyphens in verbs: 'end-user documentation' vs. 'for the end user'.
+
+### Headings  
+No multi-sentence headings or full stops at the end of a sentence in page or sentence headings. 
+Be consistent about heading levels - H1, H2, H3 - no jumping from H1 to H3 or H2 to H4.
+
+## Code and filenames  
+This section deals with adding and properly formatting code in the documentation + naming files, pages, and their derivative URLs.  
+
+### Code formatting  
+We use the basic GitHub flavor of markdown for [formatting code](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code). 
+
+For code or file names as a part of a sentence, use "\`\" before and after the word(s) that need(s) to be marked as code: Each command step can run either a shell command like `npm install`, or an executable file or script like `build.sh`. 
+In markdown this sentence looks like this: " Each command step can run either a shell command like \`npm install\`, or an executable file or script like \`build.sh\`. "
+
+Do not use code fragments in page headings or section headings.
+
+### Code blocks  
+A code example longer than a couple of words that isn’t part of a sentence/a multi-line code sample needs to be formatted as a code block according to the [GitHub markdown flavor](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+To add a code block, indent it using four (4) spaces or use 3 backticks (\`\`\`) before and after the code block.
+
+```
+Hello, world!
+```
+This probably goes without saying, but do not use code fragments in page headings or section headings.
+
+### Adding and naming new documentation pages  
+To add a new documentation page, create it as a *.md.erb file. Give it a lowercase name, separate words using underscores.
+To add the new page to the documentation sidebar on https://buildkite.com/docs, add the corresponding entry to 
+app>views>layouts>application.html.erb with a description (e.g. `"G Cloud Identity", 'integrations/sso/g-cloud-identity'` ).
+> **Note:** Ruby, which keeps the website running, interprets underscores in filenames as hyphens. So if a page is called `octopussy_cat.erb.md`, you need to add it as `octopussy-cat` to the `application.html.erb` file.     
+
+
+
+## Screenshots
+This information was aggregated by going over the existing screenshots in the documentation repo. Feel free to change or expand it.
+
+### Taking and processing screenshots  
+**Format:** PNG
```

Comment:
We should probably bullets this list so it doesn't look like this;

![image](https://user-images.githubusercontent.com/95874/110324913-e7b58180-8016-11eb-945a-99d07d87d4e3.png)